x-pack/plugins/security_solution/server/lib/detection_engine/rule_types/utils/enrichments/create_single_field_match_enrichment.ts

```diff
@@ -63,6 +71,7 @@ export const createSingleFieldMatchEnrichment: CreateFieldsMatchEnrichment = asy
       .filter((result) => result.status === 'fulfilled')
       .map((result) => (result as PromiseFulfilledResult<EnrichmentType[]>)?.value);
 
+    // search hits.
     const enrichments = flatten(enrichmentsResults);
```

Comment:
```suggestion
      .flatMap((result) => (result as PromiseFulfilledResult<EnrichmentType[]>)?.value);
```

(you'd also need to rename `enrichmentResults` to just `enrichments`, but I couldn't include that line in this comment)